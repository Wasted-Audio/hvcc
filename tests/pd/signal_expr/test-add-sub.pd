#N canvas 670 156 450 300 12;
#X obj 55 17 loadbang;
#X obj 55 352 print;
#X msg 194 154 3.1;
#X msg 140 154 2.3;
#X msg 249 154 5.5;
#X obj 55 295 snapshot~;
#X obj 55 92 t_delays;
#X obj 55 50 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 249 181 sig~;
#X obj 194 181 sig~;
#X obj 140 181 sig~;
#X obj 214 352 print;
#X obj 214 295 snapshot~;
#X obj 230 261 expr~ $v1-$v2-$v3;
#X obj 71 261 expr~ $v1+$v2+$v3;
#X connect 0 0 7 0;
#X connect 2 0 9 0;
#X connect 3 0 10 0;
#X connect 4 0 8 0;
#X connect 5 0 1 0;
#X connect 6 0 5 0;
#X connect 6 1 12 0;
#X connect 6 2 3 0;
#X connect 6 3 2 0;
#X connect 6 4 4 0;
#X connect 7 0 6 0;
#X connect 8 0 13 2;
#X connect 8 0 14 2;
#X connect 9 0 13 1;
#X connect 9 0 14 1;
#X connect 10 0 13 0;
#X connect 10 0 14 0;
#X connect 12 0 11 0;
#X connect 13 0 12 0;
#X connect 14 0 5 0;
