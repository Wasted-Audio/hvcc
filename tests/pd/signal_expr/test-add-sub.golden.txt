[@ 8.021] print: -6.3
[@ 10.021] print: 10.9
